npm/gen-root.ts

```diff
@@ -73,7 +73,7 @@ async function genServerPackage(buildDefinitions: string[]) {
 
   const postInstallScriptContent = `const version = "${packageVersion}";\n${postInstallScript}`
   const preInstallScriptContent = `const optionalDependencies = ${JSON.stringify(
-    optionalDependencies,
+    optionalDependencies
```

Comment:
we shouldn't change it as it does not relate with the issue being solved
```suggestion
    optionalDependencies,
```

Reply:
Yeah, this is certainly a formatting thing from my editor. THanks